crates/common/certificate/src/lib.rs

```diff
@@ -245,14 +244,43 @@ impl rcgen::RemoteKeyPair for RemoteKeyPair {
             .sign(msg)
             .map_err(|e| rcgen::Error::PemError(e.to_string()))
     }
+}
+
+pub struct KeyCertPair {
+    certificate: rcgen::Certificate,
+    // in rcgen 0.14 params are necessary to generate the CSR
+    params: rcgen::CertificateParams,
+    signing_key: SigningKeyWrapper,
+}
```

Comment:
Just to confirm.

Previously, the `certificate` was protected under `Zeroizing`. Now this protection has been narrowed under the `signing_key` in the case there is actually a private key in memory. This makes sense. However, can you confirm that, with version 0.14, `rcgen::Certificate` no more contains data to zeroize on drop?

Reply:
Indeed, wrapping `rcgen::Certificate` with `Zeroizing<>` generates the following error:

```
error[E0277]: the trait bound `rcgen::Certificate: zeroize::DefaultIsZeroes` is not satisfied
   --> crates/common/certificate/src/lib.rs:250:18
    |
250 |     certificate: Zeroizing<rcgen::Certificate>,
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `zeroize::DefaultIsZeroes` is not implemented for `rcgen::Certificate`
    |
```

And in 0.14 only [`rcgen::KeyPair` implements `Zeroize`](https://docs.rs/rcgen/latest/src/rcgen/lib.rs.html#882-887), which makes sense because it's the only thing holding sensitive data.